shopfloor_reception/services/reception.py

```diff
@@ -748,9 +748,21 @@ def _response_for_set_lot(self, picking, line, message=None):
             message=message,
         )
 
+    def _align_product_uom_qties(self, move):
+        qty_todo = move.product_uom_qty
+        qty_done = sum(move.move_line_ids.mapped("qty_done"))
+        rounding = move.product_id.uom_id.rounding
+        compare = float_compare(qty_done, qty_todo, precision_rounding=rounding)
+        if compare < 1:  # If qty done <= qty todo, align qty todo on move lines
+            remaining_todo = qty_todo - qty_done
+            for line in move.move_line_ids:
+                line.product_uom_qty = line.qty_done + remaining_todo
```

Comment:
Does this work if there is multiple move lines on the move ?

Reply:
yes, that's the point.
Look at the test I added: 2 users working on the same move.
Each one gets its own move line and is working in //
If we have a move with product_uom_qty = 10, each move line has qty_done = 2.
Then quantity_done on the move is 4. We're missing 6 units.
Since both move lines has qty_done 2, then the max qty should be updated to 2 + 6 == 8.
Is it clearer?

Reply:
But then on the move qty to do is 6, but on the move line it is 12 ?! 
Maybe this reasoning makes sense on the shopfloor app level (only for this scenario ?) because each move line is a presentation of the state of the move to the user (I am not sure of that). But at Odoo level it does not make sense (or for me anyway). And if I am partially right there should be a docstring on this function.
(I don't know the detail of this scenario)